src/components/index.ts

```diff
@@ -18,5 +18,4 @@ export { default as Jumbotron } from './Jumbotron';
 export { default as Table } from './Table';
 export { default as Modal } from './Modal';
 export { default as InputGroup } from './InputGroup';
-export { default as Grid } from './Grid';
-export { default as List } from './List';
```

Comment:
Should these be removed?

Reply:
those are 2 components that got merged with master this morning, it made conflicts with a bunch of components. I will try allowing them